compiler/qsc_linter/src/linter.rs

```diff
@@ -43,7 +43,7 @@ pub struct Lint {
 
 impl std::fmt::Display for Lint {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{}", self.message)
+        write!(f, "{}", self.level)
```

Comment:
So the `Display` string rendering for a Lint is just its level? That doesn't sound right.

Reply:
Yes, you are right. I changed it to something that makes more sense. Thanks!